Scripts/gen_Primary.sh

```diff
@@ -109,34 +106,36 @@ while getopts ":-:" options; do
         run)
           RUN=${!OPTIND} OPTIND=$(( $OPTIND + 1 ))
           ;;
+        setup)
+          SETUP=${!OPTIND} OPTIND=$(( $OPTIND + 1 ))
+          ;;
+#        prod)
```

Comment:
Is prod commented out because this is still in testing?

Reply:
I will remove it.
It's defined and set to true in prolog
Since prolog is used on the grid only that achieves the designed feature.

Reply:
Removed it